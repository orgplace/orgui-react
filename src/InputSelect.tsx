import classNames from 'classnames';
import React from 'react';

const OuiInputSelect: React.FC<React.SelectHTMLAttributes<HTMLSelectElement>> = (props) => {
  return (
    <select {...props} className={classNames(
      'ouiInputSelect',
      props.className,
    )}>
      {props.children}
    </select>
  );
}

export { OuiInputSelect };
