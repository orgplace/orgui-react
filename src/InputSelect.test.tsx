import React from 'react'
import { OuiInputSelect } from './InputSelect'
import renderer from 'react-test-renderer';


test('OuiInputSelect', function() {
	const component = renderer.create(<OuiInputSelect><option>A</option></OuiInputSelect>);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
