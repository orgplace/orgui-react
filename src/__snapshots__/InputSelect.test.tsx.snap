// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OuiInputSelect 1`] = `
<select
  className="ouiInputSelect"
>
  <option>
    A
  </option>
</select>
`;
